modules/jt-comparing-jenkins-tekton-concepts.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assembly:
+//
+// jenkins-tekton/migrating-from-jenkins-to-tekton.adoc
+
+[id="jt-comparing-jenkins-tekton-concepts_{context}"]
+= Comparing Jenkins and Tekton concepts
+
+toc::[]
+
+
+This section summarizes the basic terms used in Jenkins and Tekton, and compares the equivalent terms.
+
+== Jenkins terminology
+Jenkins offers declarative and scripted pipelines that are extensible using shared libraries and plugins. Some basic terms in Jenkins are as follows:
+
+* *Pipeline*: Automates the entire process of building, testing, and deploying applications, using the link:https://groovy-lang.org/[Groovy] syntax.
+* *Node*: A machine capable of orchestrating and/or executing a scripted pipeline.
+* *Stage*: A conceptually distinct subset of tasks performed in a pipeline. This block is often used by plugins or user interfaces to display status or progress of tasks.
+* **Step**: A single task that specifies the exact action to be taken, either by using a command or a script.
+
+== Tekton terminology
+Tekton uses the link:https://yaml.org/[YAML] syntax for declarative pipelines and consists of tasks. Some basic terms in Tekton are as follows:
+
+* **Pipeline**: A set of tasks that may execute serially, in parallel, or both.
```

Comment:
Optional rephrase. 

I'm not used to seeing the adverb `serially`, perhaps it might confuse users...can we say, `in a series`?  If so, consider rephrase to:
`A set of tasks that can execute in a series, in parallel, or both.`